api/client/src/main/java/org/projectnessie/client/NessieClientBuilder.java

```diff
@@ -160,6 +161,13 @@ default Set<String> names() {
   @CanIgnoreReturnValue
   NessieClientBuilder withDisableCompression(boolean disableCompression);
 
+  /**
+   * Optional, disables certificate verifications, if set to {@code true}. Can be useful for testing
+   * purposes, not recommended for production systems.
+   */
+  @CanIgnoreReturnValue
+  NessieClientBuilder withSSLNoCertificateVerification(boolean noCertificateVerification);
```

Comment:
I'd prefer a direct `boolean` property (not inversed), i.e. `withSSLCertificateVerification(false)` for disabling. Inverse it hard to read :)

Reply:
We already have `withDisableCompression`. how about `withSSLCertificateVerificationDisabled` or `withDisableSSLCertificateVerification` ?

Reply:
`withSSLCertificateVerificationDisabled` LGTM